### DISCUSSION TITLE
Do you accept pull requests for fonts?

### DISCUSSION THREAD

**Discussion by goodney** [peterhinch/micropython-nano-gui#22] [Ideas]
I found/converted a really nice size 18 font, but I imagine you don't want to fill up the font directory, so is there a better place to share fonts that look really nice on these displays?

**Comment by peterhinch** [comment]
The fonts I include are just a minimal set of examples, and I don't really want the repo to develop into a font library.

You could start a repo of fonts, but to be of use you'd need to document them, ideally with images showing their appearance. That would be a useful resource, especially if you invited others to contribute their own examples.
